search/readme.md

```diff
@@ -0,0 +1,119 @@
+# Search API
+
+## General
+Search API gives you the ability to run queries via REST (i.e. HTTP). The query language is elasticsearch Search API DSL, with certain limitations. 
```

Comment:
I would add a note saying "We highly recommend you read the restrictions listed below before beginning to use the API." linking to the section...